dataland-frontend/package.json

```diff
@@ -20,8 +20,8 @@
     "checkdependencies": "depcheck --ignores='@vue/cli-plugin-*,@vue/vue3-jest,sass-loader,ts-jest'"
   },
   "dependencies": {
-    "@formkit/themes": "^1.0.0-beta.8",
-    "@formkit/vue": "^1.0.0-beta.8",
+    "@formkit/themes": "^1.0.0-beta.9",
```

Comment:
kannst du nochmal npm updatepackagelock ausführen? Wenn ich das mit dieser package.json ausführe, bekomme ich andere Versionen in der package-lock

Reply:
ist aktualisiert

Reply:
ok